nservicebus/upgrades/6to7/index.md

```diff
@@ -28,8 +28,8 @@ Configuring Audit via the following APIs have been deprecated:
  * `HKEY_LOCAL_MACHINE\SOFTWARE\ParticularSoftware\ServiceBus\AuditQueue` registry key
  
 
-Instead use one of the following:
-BadImageFormatException
+Instead use the following:
+`endpointConfiguration.AuditProcessedMessagesTo("targetAuditQueue");`
```

Comment:
do we add a "dummy parameter" to the APIs in general or should we just use `endpointConfiguration.AuditProcessedMessagesTo(...);` or `endpointConfiguration.AuditProcessedMessagesTo` ?

Reply:
looking at some of the previous upgrade guides, using an actual parameter value seems to be quite common.